app/javascript/pages/getAssistance/housing-counselors.scss

```diff
@@ -28,4 +28,11 @@
         padding-right: var(--bloom-s2);
     }
     padding-right: 0;
+}
+
+.page-main {
+    @media (min-width: $screen-lg) {
+        @apply ml-0;
+    }
+    @apply ml-8;
```

Comment:
Do we need the margin left on smaller screens? Looking at the Figma it doesn't look like we do